src/modules/home/Home.tsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 
-import { Topbar, BasicInformation, TilesSection, RecommendedTemplates, HomeDescription, Footer } from '.';
+import Navbar from './navbar';
+import Footer from './footer';
+import { Ecosystem, Features, Search, RecommendedTemplates } from './sections';
 
 const Home = () => {
   return (
-    <div>
-      <Topbar />
-      <BasicInformation />
-      <TilesSection />
+    <>
+      <Navbar />
+      <Search />
+      <Features />
       <RecommendedTemplates />
-      <HomeDescription />
+      <Ecosystem />
       <Footer />
-    </div>
+    </>
   );
 };
```

Comment:
@polubis - some time ago you told me that we shoulnd't use ReactFragments to hold HTML structures like this one.
So my questions are:

- it is a good practice to use fragments in such cases

- why are you doing it here? does it give you something or its just for the code to be more clear and simple?

Reply:
Y its good practice but this is special case. Home page don't need any wrapper because every component inside is styled separately because of different sizes in each section. Adding here wrapper like div - gives me nothing just adds additional element. Wrapping components with html element decorated with className or id - gives styles encapsulation because of css modules so this is reason why i recommended you long time ago doing this. With that you can do things like .nameOfClass div {} and you are save